components/ui/Typography.jsx

```diff
@@ -0,0 +1,23 @@
+export function Heading({ children }) {
+  return (
+    <h1 className="font-medium text-lg sm:text-1xl xl:text-3.5xl -tracking-thin sm:-tracking-thinner xl:-tracking-thinnest leading-5.5 sm:leading-7 xl:leading-10 text-primary">
+      {children}
+    </h1>
+  );
+}
+
+export function Paragraph({ children }) {
+  return (
+    <p className="text-sm font-medium tracking-tight sm:text-base xl:text-lg sm:-tracking-thin leading-4.25 sm:leading-4.75 xl:leading-5.5 text-black">
+      {children}
+    </p>
+  );
+}
+
+export function Span({ children }) {
+  return (
+    <p className="font-medium sm:text-sm text-xs tracking-tight leading-3.75 sm:leading-4.25 text-black">
```

Comment:
It looks great! However, Span  function component should not contain paragraph(name should reflect functionallity) you may either change the name of the component or  use span instead of paragraph.